README.md

```diff
@@ -24,9 +24,12 @@ For when you want to use image registries as a way to push/pull volume data.
 
 Usage:
 
-`vackup export VOLUME FILE`
+`vackup export [OPTIONS] VOLUME FILE`
 Creates a gzip'ed tarball in current directory from a volume
 
+- `--owner=UID[:GID]`
+  Change the ownership of the exported archive on Linux (may not have any effect on macOS and BSD)
```

Comment:
No reason to mention BSD. The "not have an effect" is related to Docker Desktop, which doesn't run on BSD. (fix in readme and script)

Reply:
yes remove plz

Reply:
Is this ok:

`--owner=UID[:GID]       Change the ownership of the exported archive on Linux (won't have any effect on macOS)`